diagnostic_feedback/quiz_result.py

```diff
@@ -0,0 +1,94 @@
+import collections
```

Comment:
This import is not used. Please remove this and the other unused imports reported by pylint:

```
************* Module diagnostic_feedback.quiz_result
W:  1, 0: Unused import collections (unused-import)
************* Module diagnostic_feedback.tests.unit.base_test
W:  1, 0: Unused import logging (unused-import)
************* Module diagnostic_feedback.tests.unit.test_diagnostic_feedback
W:  6, 0: Unused assert_true imported from nose.tools (unused-import)
W:  6, 0: Unused assert_in imported from nose.tools (unused-import)
W:  6, 0: Unused assert_false imported from nose.tools (unused-import)
************* Module diagnostic_feedback.validators.validator
W:  1, 0: Unused import BaseValidator (unused-import)
```